content/en/apps/reference/app-settings/transitions.md

```diff
@@ -433,6 +433,11 @@ Configuration is stored in the `muting` field of `app_settings.json`.
 | `validations` | List of form fields validations. All mute & unmute forms will be subjected to these validation rules. Invalid forms will not trigger muting/unmuting actions. Optional. |
 | `messages` | List of tasks/errors that will be created, determined by `event_type`. Optional. |
 
+{{% alert title="Note" %}}
+Contact forms cannot trigger muting or unmutting, but any `data_record` that has a `form` property can.
```

Comment:
@michaelkohn can we add a link to what a `data_record` is or what constitutes a `data_record`?

Reply:
OOh good suggestion. I linked to the "Report" documentation in the db-schema section since they are data_records. I could really find anything else describing what a data_record is. 

What do you think?